diff --git a/build/useScreens.js b/build/useScreens.js
--- a/build/useScreens.js
+++ b/build/useScreens.js
@@ -126,8 +126,11 @@ function useScreens(context, layout, options, order) {
     }), [sorted, protectedScreens]);
 }
-function fromImport(value, { ErrorBoundary, ...component }) {
-    // If possible, add a more helpful display name for the component stack to improve debugging of React errors such as `Text strings must be rendered within a <Text> component.`.
-    if (component?.default && __DEV__) {
+function fromImport(value, importedModule) {
+    if (importedModule == null) {
+        return { default: EmptyRoute_1.EmptyRoute };
+    }
+    const { ErrorBoundary, ...component } = importedModule;
+    // If possible, add a more helpful display name for the component stack to improve debugging of React errors such as `Text strings must be rendered within a <Text> component.`.
+    if (component?.default && __DEV__) {
